docs/reference/configuration/keywords.rst

```diff
@@ -32,7 +32,7 @@ Keyword name                                                            Required
 :ref:`ECLBASE <eclbase>`                                                NO                                                                      Define a name for the ECLIPSE simulations.
 :ref:`STD_CUTOFF <std_cutoff>`                                          NO                                      1e-6                            Determines the threshold for ensemble variation in a measurement
 :ref:`ENKF_ALPHA <enkf_alpha>`                                          NO                                      3.0                             Parameter controlling outlier behaviour in EnKF algorithm
-:ref:`ENKF_TRUNCATION <enkf_truncation>`                                NO                                      0.99                            Cutoff used on singular value spectrum
+:ref:`ENKF_TRUNCATION <enkf_truncation>`                                NO                                      0.98                            Cutoff used on singular value spectrum
```

Comment:
I guess you double-checked that `The default value of ENKF_TRUNCATION is 0.98` was not an old typo before changing  this to 0.98 :sweat_smile:

Reply:
I checked the codebase, but also now inspected the refactor that was done prior to this. They all have `0.98` as the default value.